src/components/App/App.js

```diff
@@ -1,12 +1,8 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PageHeader from '../PageHeader/PageHeader';
 
 import './App.css';
 
-class App extends Component {
```

Comment:
remind me that we can utilize 'useState' in functional components instead of 'setState' in class components